cdci_data_analysis/analysis/parameters.py

```diff
@@ -382,30 +393,34 @@ def value(self):
 
     @value.setter
     def value(self, v):
-        if v is not None and v!='':
+        if v is not None and v != '':
             self.check_float_value(v, name=self.name, units=self.units)
             self._v = np.float(v)
 
         else:
             self._v=None
 
+    def get_value_in_default_format(self, value):
+        return float(value)
+
     @staticmethod
     def check_float_value(value, units=None, name=None):
         #print('check type of ',name,'value', value, 'type',type(value))
-        if value is None or value=='':
+        if value is None or value == '':
             pass
         else:
             try:
-              value=ast.literal_eval(value)
+                value = ast.literal_eval(value)
             except:
                 pass
-            value=np.float(value)
-            if type(value) == int or type(value) == np.int:
-                pass
-            elif type(value) == float or type(value) == np.float:
-                pass
-            else:
-                raise RuntimeError('type of ', name, 'not valid', type(value))
+            try:
+                value = float(value)
+                if type(value) == float or type(value) == np.float:
```

Comment:
`value` was just cast to `float`. This check is useless.

Reply:
indeed, I realized it and removed it